src/Test/TestCombat.java

```diff
@@ -54,6 +58,7 @@ private boolean testUnitDeath()
 
     // Clean up
     testMap.removeUnit(mechA);
+    CombatEngine.modifiers.remove(mod);
```

Comment:
It's not necessary in normal gameplay. I just did that cleanup to make sure there weren't weird sideeffects.